report_thesis/src/sections/problem_definition.tex

```diff
@@ -0,0 +1,54 @@
+
+In this work, we focus on dimensionality reduction and model selection over outlier removal.
+This is justified by the low incidence of outliers in the ChemCam \gls{libs} calibration dataset, as reported in \cite{p9_paper}.
+Dimensionality reduction is crucial for managing the high-dimensional nature of \gls{libs} data. % TODO: There are lots of related works which explore DR in LIBS data. We can back this up with citations.
+Furthermore, model selection shows promise in addressing the limitations of the current MOC pipeline, as it allows for the exploration of a wider range of algorithms, potentially leading to improved performance.
+We showed that advanced ensemble methods, such as \gls{gbr}, and deep learning models, such as \gls{ann}, have the potential to outperform the current MOC pipeline.
+Methods are selected based on their promise in handling high-dimensional, non-linear data. Ideally, the selected methods should also be feasible for small datasets, a common scenario in \gls{libs} analyses. 
+In order to address the aforementioned challenges, we propose to explore advanced ensemble methods and deep learning models, which have shown promise in handling high-dimensional, non-linear data.
+
+It is necessary to establish metrics to evaluate the performance of the models.
+In \cite{p9_paper}, we proposed to use the \gls{rmse} as a proxy for accuracy.
+
+\gls{rmse} is given by:
+
+\begin{equation}
+    RMSE = \sqrt{\frac{1}{n} \sum_{i=1}^{n} (y_i - \hat{y}_i)^2}
+\end{equation}
+
+where $y_i$ represents the actual values, $\hat{y}_i$ the predicted values, and $n$ the number of observations.
+
+To address robustness, we propose considering the standard deviation of prediction errors across each oxide and test instance, defined as:
+
+\begin{equation}
+    \sigma_{error} = \sqrt{\frac{1}{n-1} \sum_{i=1}^{n} (e_i - \bar{e})^2}
+\end{equation}
+
+where $e_i = y_i - \hat{y}_i$ and $\bar{e}$ is the mean error.
+
+In order to narrow down the scope of our research, we set the following constraints:
+\begin{itemize}
+    \item Prioritize normalization across individual spectrometers' wavelength ranges over full-spectrum normalization.
+    \item Focus on techniques proven effective for non-linear, high-dimensional data, even outside the \gls{libs} context.
+    \item Ensure methods are feasible for small datasets.
+\end{itemize}
+
+In \cite{p9_paper}, we used both full-spectrum normalization (Norm 1) and normalization across individual spectrometers' wavelength ranges (Norm 3).
+However, in this work, we opt for normalizing across individual spectrometers' wavelength ranges.
+This decision is guided by the operational parameters of SuperCam\cite{andersonPostLandingMajorElements2022}, where they do not normalize across the entire spectrum, but rather across individual spectrometers' wavelength ranges.
```

Comment:
What is meant by "operational parameters"? Should maybe elaborate a bit since this is a fairly major decision.

Reply:
Clarified